types/react-leaflet/v1/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+	"extends": "dtslint/dt.json",
+	"rules": {
+		// TODO
```

Comment:
I'm guessing this is just carried over from the version change, and that you didn't add it for some other reason?

Reply:
Correct, that's from here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/09b9ef4ff0932791ff758e5c4c5baed0409f764f/types/react-leaflet/tslint.json#L4